addons/web/static/src/views/fields/color/color_field.js

```diff
@@ -1,24 +1,35 @@
 import { Component } from "@odoo/owl";
 import { registry } from "@web/core/registry";
 import { standardFieldProps } from "../standard_field_props";
+import { exprToBoolean } from "@web/core/utils/strings";
 
 export class ColorField extends Component {
     static template = "web.ColorField";
     static props = {
         ...standardFieldProps,
+        autoSave: { type: Boolean, optional: true },
```

Comment:
```suggestion
        autosave: { type: Boolean, optional: true },
```

by consistency with other fields